lib/cocoapods/installer/xcode/pods_project_generator/app_host_installer.rb

```diff
@@ -51,12 +60,13 @@ def install!
               configuration.build_settings['CODE_SIGN_IDENTITY'] = '' if platform == :osx
               configuration.build_settings['CURRENT_PROJECT_VERSION'] = '1'
             end
-            Pod::Generator::AppTargetHelper.add_app_host_main_file(project, app_host_target, platform_name, name)
-            Pod::Generator::AppTargetHelper.add_launchscreen_storyboard(project, app_host_target, name) if platform == :ios
+
+            Pod::Generator::AppTargetHelper.add_app_host_main_file(project, app_host_target, platform_name, group, name)
+            Pod::Generator::AppTargetHelper.add_launchscreen_storyboard(project, app_host_target, group, name) if platform == :ios
             additional_entries = platform == :ios ? ADDITIONAL_IOS_INFO_PLIST_ENTRIES : {}
             create_info_plist_file_with_sandbox(sandbox, app_host_info_plist_path, app_host_target, '1.0.0', platform,
                                                 :appl, additional_entries)
-            project[name].new_file(app_host_info_plist_path)
+            group.new_file(app_host_info_plist_path)
```

Comment:
Should we be doing something like:
```ruby
        group = project[name] || project.new_group(name, name)
```

Similar to what `AppTargetHelper` does in which we ensure the group exists before we use it, either because it existed before or because we just created it.

Reply:
Added something like this.

Reply:
But where we pass in the group into the `app_host_installer.rb`